pyproject.toml

```diff
@@ -62,3 +62,7 @@ markers = [
     "db_restore_dump(dump): mark the db dump to restore. To be used in conjunction with the `migrate_db_from` fixture."
 ]
 log_format = "%(asctime)s.%(msecs)03d %(levelname)s %(message)s"
+
+[tool.coverage.run]
+parallel = true
```

Comment:
Why set the parallel option to True? The pytest-cov documentation mentions that it overrides this option (See: https://pytest-cov.readthedocs.io/en/latest/config.html)

Reply:
ok, was not quite sure. 

I did it in case the tooling loses track of the config file because of changes to cwd. 

But I can also remove it

```suggestion
```